pkg/orchestration/wiring/kubeingress/plugin.go

```diff
@@ -51,7 +52,7 @@ func WireUp(resource *orch_v1.StateResource, context *wiringplugin.WiringContext
 		return nil, false, errors.Errorf("invalid resource type: %q", resource.Type)
 	}
 
-	deploymentSpec, retriable, err := extractKubeComputeDependency(context.Dependencies)
+	deploymentSpec, retriable, err := extractKubeComputeDependency(&context.Dependencies)
```

Comment:
IMO it's a bad idea to pass a variable-sized unbounded object by reference unless the intent is to mutate it. Copying is bad.

Reply:
This code has no idea how large "context.Dependencies" is - it's owned by another module. If it contains fifty members, we're going to copy fifty members.

That said, I've changed everything to pass-by-value.